frontend/src/utils/rolesCheck.tsx

```diff
@@ -0,0 +1,11 @@
+export const rolesCheck = (userRoles: Array<string>, requiredRoles: Array<string>) => {
```

Comment:
Nazwa parametru `requiredRoles` sugeruje, że są to wymagane role, podczas gdy logika w tej metodzie sprawdza czy wystarczy jedna rola spośród przekazanych.